docs/validators/setup.md

```diff
@@ -0,0 +1,87 @@
+---
+sidebar_position: 2
+---
+
+# Setting Up a Node
+
+In this guide we will be setting up a ZetaChain Mainnet Beta node. For a testnet
+node, follow the same instructions, but replace some values (like chain ID and
+seed nodes) with the ones provided in corresponding tables.
+
+## Install the ZetaChain Node Binary
+
+First, download the binary and give it executable permissions. Make sure to
+replace the version (`v13.0.0`) with the latest version passed in a
+[governance software upgrade proposal](/validators/proposals) and the platform
+(`-darwin-amd64`) with the one you need.
+
+```
+wget https://github.com/zeta-chain/node/releases/download/v13.0.0/zetacored-darwin-amd64 -O /usr/local/bin/zetacored
```

Comment:
This will not work for Mainnet which is skipping v13. Today it runs v12.2.0. 

Athens is running v13.0.0 today but will run v14.0.0 early next week.

Reply:
Yes, but there is a note asking readers to replace the version with the right one from the table:

https://github.com/zeta-chain/docs/blob/8c500d21460d507185221dec34b9a6343dc1846c/docs/validators/setup.md?plain=1#L13-L14

I think we can switch to determining the right version programmatically, but for now I think this could work.

Reply:
If we could change it to "v14" it would be more accurate. "v14" activates today for Athens. "v13" will never be used in Mainnet.

Reply:
Agreed. Lets set it to v14 for now since that's about to go live.

Reply:
It looks like this still needs to be updated away from referencing "v13"

Reply:
Replaced "v13" with a placeholder: a46f1b1